spec/models/video_spec.rb

```diff
@@ -37,4 +38,14 @@
       expect(Video.search_by_title("")).to eq([])
     end
   end
+
+  describe "#average_rating" do
+    it "calculates the average rating of a video (from its reviews) rounded to 1 decimal place" do
+      video = Fabricate(:video)
+      video.reviews << Fabricate(:review, rating: 5)
```

Comment:
You can create `review` directly under `video` like this `Fabricate(:review, rating: 5, video: video)`